src/components/card/FlipCard.js

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from 'styled-components/macro'
+import { useState } from 'react'
+import mainBackground from '../../assets/images/mainBackground.jpg'
+
+export default function FlipCard(props) {
+  const [isCardFlipped, setIsCardFlipped] = useState(false)
+  const isFreeOfCharge = props.freeOfCharge
```

Comment:
It is okay to do it like this, but I would maybe change it in your data structure already?